lib/std/wrapnils.nim

```diff
@@ -1,9 +1,20 @@
-## This module allows chains of field-access and indexing where the LHS can be nil.
-## This simplifies code by reducing need for if-else branches around intermediate values
-## that may be nil.
+## This module allows evaluating expressions safely against the following conditions:
+## * nil dereferences
+## * field accesses with incorrect discriminant in case objects
+##
+## A default value is returned in those cases.
+## This simplifies code by reducing need for if-else branches.
```

Comment:
How is the default value determined? Is it just the default value of the return type? Either way, this should be documented.